test/service/test_manifest.py

```diff
@@ -1188,8 +1189,32 @@ def test_full_metadata_missing_fields(self):
         intersection = reduce(set.intersection, fieldnames)
         symmetric_diff = reduce(set.symmetric_difference, fieldnames)
 
-        self.assertGreater(len(intersection), 0)
-        self.assertGreater(len(symmetric_diff), 0)
+    @manifest_test
+    def test_manifest_bulk_download(self):
+        self.maxDiff = None
+        bundle_fqid = BundleFQID('f79257a7-dfc6-46d6-ae00-ba4b25313c10', '2018-09-14T133314.453337Z')
+        self._index_canned_bundle(bundle_fqid)
+        filters = {'fileFormat': {'is': ['pdf']}}
+        response = self._get_manifest(ManifestFormat.bulk_download, filters)
+        self.assertEqual(200, response.status_code, 'Unable to download manifest')
+        lines = response.content.decode('utf-8').splitlines()
+        self.assertIn('--create-dirs', lines)
+        self.assertIn('--compressed', lines)
+        file_uuids, file_names = [], []
+        for line in lines:
+            if line.startswith('url='):  # url="https://.../repo/files/74897eb7-0701-4e4f-9e6b-8b9521b2816b"
```

Comment:
This is too much post-processing of the response. This is very simple format and the index content is known. This can be a much more literal assertion like some of the other tests in this class.